robot/impl/local_robot_test.go

```diff
@@ -4284,3 +4284,71 @@ func TestMaintenanceConfig(t *testing.T) {
 		test.That(t, sensorUnBlocked, test.ShouldNotBeNil)
 	})
 }
+
+func TestRemovingOfflineRemote(t *testing.T) {
```

Comment:
can we have a test where the robot starts with a remote that points at the wrong address, fails to connect and then we remove the remote? that does go through a different path so I'd be interested as to what happens there.